tests/ui/methods.rs

```diff
@@ -36,10 +37,137 @@ use option_helpers::IteratorFalsePositives;
 pub struct T;
 
 impl T {
+    // *******************************************
+    // complete trait method list, should lint all
+    // *******************************************
     pub fn add(self, other: T) -> T {
```

Comment:
Please move out those test cases to their own file, since the `methods.rs` test file is already overloaded.

Reply:
moved in should_impl_trait.rs